swig/lightgbmlib.i

```diff
@@ -156,7 +156,13 @@
 
       // get the size, indices and values
       auto indices = (jintArray)jenv->CallObjectMethod(objSparseVec, sparseVectorIndices);
+      if (jenv->ExceptionCheck()) {
+	return -1;
```

Comment:
@imatiach-msft  Please don't use tabs
https://github.com/microsoft/LightGBM/blob/023749239ebf390b5d56e08f227a013cbd972eb0/.editorconfig#L7-L8

Reply:
done!  @StrikerRUS sorry about that!  I really should figure out how to configure my emacs to not use tabs when I have the time...